pandas/core/base.py

```diff
@@ -521,9 +521,14 @@ def searchsorted(self, key, side='left'):
     def _make_str_accessor(self):
         from pandas.core.series import Series
         from pandas.core.index import Index
-        if isinstance(self, Series) and not com.is_object_dtype(self.dtype):
-            # this really should exclude all series with any non-string values,
-            # but that isn't practical for performance reasons until we have a
+        if isinstance(self, Series) and not(
+                    (com.is_categorical_dtype(self.dtype) and
+                     com.is_object_dtype(self.values.categories)) or
```

Comment:
Can do, but as I understand the code, I made sure that it is a categorical before using the categories on `values`: `com.is_categorical_dtype(self.dtype) and com.is_object_dtype(self.values.categories)`